.github/workflows/tests.yml

```diff
@@ -30,8 +30,5 @@ jobs:
       - name: Prepare for Integration Tests
         run: npm run demo:build
 
-      - name: Install playwright browsers
```

Comment:
Does this mean that some other thing installs chromium? The change description and this removal seem to say opposite things at first glance

Reply:
The `@playwright/browser-chromium` dep in package.json installs chromium now